docs/proposals/scheduling/multi-podtemplate-support/multiple-pod-template-support.md

```diff
@@ -1,141 +1,129 @@
 ---
-title: CRD Component Scheduler Estimation
+title: Multiple Pod Templates Scheduling
 authors:
   - "@mszacillo"
```

Comment:
@RainbowMango You should definitely add yourself as an author here. :)

Reply:
:)
OK.

I'm surprised you get up so early, I remember you are in the timezone UTC-4.

Reply:
I'm working from Poland this week, so my working ours are a bit earlier now haha